src/transactions/test/InvokeHostFunctionTests.cpp

```diff
@@ -392,30 +392,65 @@ TEST_CASE("contract storage", "[tx][soroban]")
                          {contractDataKey(contractID, makeSymbol(key))}, true);
     };
 
-    put("key1", 0);
-    put("key2", 21);
+    SECTION("success")
```

Comment:
I wasn't sure about adding sections here as every section would cause the test to re-run and hence the storage will be re-created every time, so we won't cover modification.

Reply:
Got it. I've moved your tests to its own section so that they are not resetting the storage. 

For mine which overrides the limits, I've kept it in a separate section so they won't carry over which is nicer.